LICENSE.md

```diff
@@ -93,21 +90,21 @@ A Derivative must contain changes which a reasonably informed person would consi
 The User may distribute a Derivative if the following conditions are met:
  - The Derivative does not generate revenue.
  - The Derivative provides credit to the Author.
- - The Derivative complies with the Mod license.
+ - The Derivative, as a whole, is licensed under this license for anyone who comes into possession of a copy.
+ - The User provides means to obtain the Derivative's source code at no charge.
```

Comment:
Requiring a distributor to provide the source is outside the scope of this section, same as for the distribution section above. Stating that the Derivative must comply with the Mod license is sufficient, if the Mod license includes a requirement for visible source. I haven't figured out exactly what the best way to do that, but requiring source code visibility here isn't the right place.